### PR TITLE
Nerf scrapbot on asteroids

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Added spawn placeholder animation for scrap bot spawning. Actual crime against spriting.
- Nerfed spawn rates and movement speed for laser bot.

## Why / Balance
BaLaNcE

## How to test
Locate scrap 'roid, mine, observe.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- tweak: Slightly nerfed asteroid scrapbots.

### CODE REVIEW

**Review by dvir001 — APPROVED** [review]

**Comment by whatston3** [comment]
Threw the Jittering component on one of the spawners just to see how it'd look:

https://github.com/user-attachments/assets/a6f4c7eb-1029-45a6-8c01-7d1dcb6f94b3

Not awful, if the amplitude and frequency were reduced, might look neat.

Fine as-is though.

**Comment by ErhardSteinhauer** [comment]
> Threw the Jittering component on one of the spawners just to see how it'd look:
> 
> Not awful, if the amplitude and frequency were reduced, might look neat.
> 
> Fine as-is though.

That's actually a very clever way to conceal dogwater quality of sprites and make 'em stand out. I love it.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml [hunk 1]
```diff
@@ -274,12 +274,12 @@
       color: "#990000"
   - type: RandomSpawner
     rarePrototypes:
-    - MobRogueSiliconScrapThrustersRangedAsteroid
-    rareChance: 0.1
+    - MobSpawnScrapbotLaser #MobRogueSiliconScrapThrustersRangedAsteroid
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml:277]:
```suggestion
    - MobSpawnScrapbotLaser
```
Are the comments really meaningful here?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml [hunk 2]
```diff
@@ -274,12 +274,12 @@
...
+    rareChance: 0.05
     prototypes:
-    - MobRogueSiliconScrapAsteroid
-    - MobRogueSiliconScrapThreadsAsteroid
-    - MobRogueSiliconScrapThrustersAsteroid
+    - MobSpawnScrapbot #MobRogueSiliconScrapAsteroid
+    - MobSpawnScrapbotThreads #MobRogueSiliconScrapThreadsAsteroid
+    - MobSpawnScrapbotThrusters #MobRogueSiliconScrapThrustersAsteroid
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml:282]:
```suggestion
    - MobSpawnScrapbot
    - MobSpawnScrapbotThreads
    - MobSpawnScrapbotThrusters
```
Are the comments really meaningful here?

**Review by whatston3 — COMMENTED** [review]
Might delete the comments on some of these lines, otherwise seems alright.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml (hunk 1 above)
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/salvage.yml:277]: Something-something, uhh

**Review by whatston3 — APPROVED** [review]

**Pull request merged**